kolibri/plugins/management/assets/src/vue/modal/index.vue

```diff
@@ -82,28 +78,26 @@
       if (this.disableClose) {
         this.$off('close');
       }
+
+      this.$emit('open');
     },
     events: {
       open() {
-        this.visible = true;
         this.lastFocus = document.activeElement;
         // Need to wait for DOM to update asynchronously, then get the modal element
-        vue.nextTick(() => {
-          this.focusModal();
-          // pass in a function, not a function call.
-          window.addEventListener('blur', this.focusElementTest, true);
-        });
+        this.focusModal();
+        // pass in a function, not a function call.
+        window.addEventListener('blur', this.focusElementTest, true);
+        window.addEventListener('scroll', (event) => event.preventDefault());
```

Comment:
No, I have the listener happen when the modal appears. You think this is a bad idea? Should I just put it under `ready`?